test/contracts/consumer/fabric8auth/api_status.go

```diff
@@ -0,0 +1,55 @@
+package fabric8auth
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"testing"
+
+	"github.com/fabric8-services/fabric8-auth/test/contracts/model"
+	"github.com/pact-foundation/pact-go/dsl"
+)
+
+// AuthAPIStatus defines contract of /api/status endpoint
+func AuthAPIStatus(t *testing.T, pact *dsl.Pact) {
+
+	log.Printf("Invoking AuthAPIStatus now\n")
+
+	// Pass in test case
+	var test = func() error {
+		u := fmt.Sprintf("http://localhost:%d/api/status", pact.Server.Port)
+		req, err := http.NewRequest("GET", u, nil)
+
+		req.Header.Set("Content-Type", "application/json")
+		if err != nil {
+			return err
+		}
+
+		_, err = http.DefaultClient.Do(req)
+		if err != nil {
+			return err
+		}
+		return err
+	}
+
+	// Set up our expected interactions.
+	pact.
+		AddInteraction().
+		Given("Auth service is up and running.").
+		UponReceiving("A request to get status").
+		WithRequest(dsl.Request{
+			Method:  "GET",
+			Path:    dsl.String("/api/status"),
+			Headers: dsl.MapMatcher{"Content-Type": dsl.String("application/json")},
+		}).
+		WillRespondWith(dsl.Response{
+			Status:  200,
+			Headers: dsl.MapMatcher{"Content-Type": dsl.String("application/vnd.status+json")},
+			Body:    dsl.Match(model.APIStatusMessage{}),
+		})
+
+	// Verify
+	if err := pact.Verify(test); err != nil {
+		log.Fatalf("Error on Verify: %v", err)
```

Comment:
If you print or log an error, please also use `%+v` in order to print the stack trace (if available) that leads up to the error. See https://godoc.org/github.com/pkg/errors#hdr-Formatted_printing_of_errors:

```
%s    print the error. If the error has a Cause it will be
      printed recursively.
%v    see %s
%+v   extended format. Each Frame of the error's StackTrace will
      be printed in detail.
```

Reply:
Thanks for the tip - fixed.